scripts/scr_company_view/scr_company_view.gml

```diff
@@ -1,43 +1,21 @@
-function scr_company_view(argument0) {
+function scr_company_view(company) {
 
-	var mans, onceh, company, bad, squads, squad_typ, squad_loc, squad_members;
-	gogogo=0;
-	mans=0;
-	vehicles=0;
-	last_man=0;
-	last_vehicle=0;
-	bad=0;
-	squads=0;
-	squad_typ="";
-	squad_loc=0;
-	squad_members=0;
+	var v, i, mans, onceh,bad, squads, squad_type, squad_loc, squad_members, unit, unit_loc;
+	gogogo=0;mans=0;vehicles=0;last_man=0;last_vehicle=0;
+	mans=0;vehicles=0;v=0;i=-1;bad=0;
+	squads=0;squad_type="";squad_loc=0;squad_members=0;
+	var command_roles= [obj_ini.role[100,5],obj_ini.role[100,14],obj_ini.role[100,15],obj_ini.role[100,16],ma_role[v]!=obj_ini.role[100,17],"Standard Bearer","Company Champion","Codiciery","Lexicanum"];
 
 	// v: check number
 	// mans: number of mans that a hit has gotten
 	// Calculates the temporary variables to be displayed as marines in the individual company screens
 
-	for (var i = 1; i <= 499; i++){
-	    man[i]="";
-		ide[i]=0;
-		man_sel[i]=0;
-		ma_lid[i]=0;
-		ma_wid[i]=0;
-		ma_bio[i]=0;
-	    ma_race[i]=0;
-		ma_loc[i]="";
-		ma_name[i]="";
-		ma_role[i]="";
-		ma_gear[i]="";
-		ma_mobi[i]="";
-		ma_wep1[i]="";
-	    ma_wep2[i]="";
-		ma_armour[i]="";
-		ma_health[i]=100;
-		ma_chaos[i]=0;
-		ma_exp[i]=0;
-		ma_promote[i]=0;
-		ma_god[i]=0;
-	}
+	repeat(499){i+=1;
```

Comment:
why `repeat` instead of `for(var i = 1...`?

Reply:
this seems to be a duplication of the earlier repeat i think coming from a merge conflict with one of the refactors. Will fix

Reply:
made into correct syntax